tests/conftest.py

```diff
@@ -1,18 +1,36 @@
 import pytest
+from _pytest.fixtures import FixtureRequest
 from flask import Flask
 from flask.testing import FlaskClient
 
 from time_tracker_api import create_app
 
+CONFIGURATIONS = ['AzureSQLDatabaseDevelopTestConfig']
 
-@pytest.fixture(scope='session')
-def app() -> Flask:
+
+@pytest.fixture(scope='session', params=CONFIGURATIONS)
+def app(request: FixtureRequest) -> Flask:
     """An instance of the app for tests"""
-    return create_app()
+    return create_app("time_tracker_api.config.%s" % request.param)
 
 
 @pytest.fixture
 def client(app: Flask) -> FlaskClient:
     """A test client for the app."""
     with app.test_client() as c:
         return c
+
+
+@pytest.fixture(scope="module")
```

Comment:
What's the intention of this test?

Reply:
This is not a test, these are fixtures, which are basically a factory where objects are created to be injected in the tests. It works basically as an IoC mechanism.

Reply:
great!!!